src/app/Http/Requests/FinancialAccounts/CreateFinancialAccountRequest.php

```diff
@@ -23,10 +23,13 @@ public function authorize()
      */
     public function rules()
     {
-        //note - for the 'max' to work, double quotes are apparently necessary (or the pipe delimiter notation)
```

Comment:
ver či nie, na mojom počítači mi to bez double quotes naozaj padalo ako neznáme validačné pravidlo :D (to je len taká anekdota pre zaujímavosť, nič netreba meniť)

Reply:
môže byť...ja som to zmenil, lebo mne to fungovalo aj s jednoduchými apostrofmi